fbw-common/src/systems/oans/OansMapProjection.ts

```diff
@@ -26,4 +26,25 @@ export class OansMapProjection {
 
     return out;
   }
+
+  /**
+   * Reverse transform: airport local coordinates (x east, y north) back to global WGS84 coordinates.
+   *
+   * @param airportPos Airport reference coordinates (origin of local system)
+   * @param local Airport local coordinates [x_east_m, y_north_m]
+   * @param out Output argument: Write global coordinates here
+   */
+  public static airportToGlobalCoordinates(airportPos: Coordinates, local: Position): Coordinates {
+    const nmToMeters = 1_000 / 0.539957;
```

Comment:
You can just do the exact conversion.
```suggestion
    const nmToMeters = 1852;
```

Reply:
Indeed, vibe coded too hard here